test/node/decimal128.test.ts

```diff
@@ -107,8 +109,8 @@ describe('Decimal128', function () {
 
   it('fromString simple', function (done) {
     // Create decimal from string value 1
-    var result = Decimal128.fromString('1');
-    var bytes = Buffer.from(
+    let result = Decimal128.fromString('1');
```

Comment:
Same as above comments... Instead of reassigning `result` can we break out into multiple `it` blocks with their own assertions? (Just making comments here since there's a test refactor my opinion is we might as well clean it up further.)